comms-commands/orb.yml

```diff
@@ -15,25 +15,14 @@ jobs:
         description: "This can be either one of (prd, uat or lt)"
         type: enum
         enum: [ "prd", "uat", "lt" ]
```

Comment:
I think it would be better to remove environment and keep the paths. It would be more context agnostic that way.